certbot/CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
   removed in a future version of Certbot.
 * Packaged tests for all Certbot components besides josepy were moved inside
   the `_internal/tests` module.
+* Lineage name validity is performed for new lineages. `--cert-name` may no longer contain
+  filepath separators (i.e. `/` or `\\`, depending on the platform).
```

Comment:
I don't think we need to escape this here.
```suggestion
  filepath separators (i.e. `/` or `\`, depending on the platform).
```
Also, can you merge master into this branch and then move this changelog entry to the new 2.6.0 section? We really should fix https://github.com/certbot/certbot/issues/8272.